### PR TITLE
Draft for Cavas in Pixi.js

### PR DESCRIPTION
**Pull request by casperstorm**
This is WIP, so don't merge it yet. This is soley for discussing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@inlet/react-pixi": "^0.3.0",
```
Comment (casperstorm) [inline package.json]: This seems like the proper module to use when we want Pixi.js and React at the same time.

Comment (jstoone) [reply package.json]: Never thought of using it in this way. Could be an interesting workflow. 👍

**Review by jstoone — APPROVED** [review]
Looks like a solid boilerplate. Appreciate the storybook setup. 👌

**Comment by casperstorm** [comment]
I now have a `Canvas` which are able to draw `Sprites` which each has a loop-function.
Before I do more, we need to start working on https://github.com/boost-me/engine/issues/3.

Maybe @egeriis or @steffenagger would like to start on that?

File: src/components/canvas/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <FireCircle
+              x={510}
+              y={580}
+              radius={23}
+              fill={0xff5b2e}
+              alpha={0.85}
+              app={app}
+            />
+          )}
+        </AppContext.Consumer>
```
Comment (egeriis) [inline src/components/canvas/index.tsx:53]: Why multiple `AppContext.Consumer`s?

File: src/components/canvas/index.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  public render() {
+    const { height, width } = this.props
+    return (
+      <Stage width={width} height={height}>
+        <Sprite
+          width={width}
+          height={height}
+          texture={PIXI.Texture.fromImage(mirage)}
+        />
+        <AppContext.Consumer>
```
Comment (egeriis) [inline src/components/canvas/index.tsx:27]:
```suggestion
        <AppContext.Consumer>
          {app => (
            <Fragment>
              <PlayerSprite app={app} />
              <SmokeCircle
                x={395}
                y={500}
                radius={23}
                fill={0xb5b5b5}
                alpha={0.85}
                app={app}
              />
              <FireCircle
                x={510}
                y={580}
                radius={23}
                fill={0xff5b2e}
                alpha={0.85}
                app={app}
              />
            </Fragment>
          )}
        </AppContext.Consumer>
```

**Comment by casperstorm** [comment]
Closed in favor for #19.

**Pull request closed without merging**